core/services/ardupilot_manager/mavlink_proxy/main.py

```diff
@@ -38,18 +38,21 @@
     parser.add_argument(
         "--tool",
         dest="tool",
-        type=str,
-        default=interfaces[0].name(),
         choices=[interface.name() for interface in interfaces],
-        help="Selected the desired tool to use, the default will be the first one available.",
+        help="Selected the desired tool to use, this will override the one in the settings file."
+        + " The default will be the first one available.",
```

Comment:
```suggestion
        help="Select the desired tool to use, this will override the one in the settings file."
        + " The default will be the first one available.",
```